prismic-model/src/exhibition-guides.ts

```diff
@@ -0,0 +1,44 @@
+import { CustomType } from './types/CustomType';
+import title from './parts/title';
+import link from './parts/link';
+import list from './parts/list';
+import image from './parts/image';
+import structuredText from './parts/structured-text';
+import embed from './parts/embed';
+import contributorsWithTitle from './parts/contributorsWithTitle';
+import number from './parts/number';
+
+const exhibitionGuides: CustomType = {
+  id: 'exhibition-guides',
+  label: 'Exhibition Guide',
```

Comment:
This should have a lower case 'g' to be consistent with the other labels in Prismic